### PR TITLE
fix: Automated test execution access to net portal network proxy

### PR DESCRIPTION
**Pull request by kakj-go**
#### What this PR does / why we need it:
Automated test execution access to net portal network proxy

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](https://erda.cloud/erda/dop/projects/387/issues/gantt?filter__urlQuery=eyJpdGVyYXRpb25JRHMiOlsxMDkwLDExMTUsMTA2OSw4ODMsODgxLDc3Ml0sImFzc2lnbmVlIjpbIjEwMDA1NjAiXX0%3D&id=285083&iterationID=1090&pId=0&type=BUG)

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |        Fix the error of automatic test execution, and netprotocol is not found      |
| 🇨🇳 中文    |       修复自动化测试执行报错没有找到 netprotal       |

### CODE REVIEW

**Comment by kakj-go** [comment]
scene test image:

image

run step test image:

image

image

**Review by sfwn — CHANGES_REQUESTED** [review]

File: apistructs/cluster_info.go [hunk 1]
```diff
@@ -45,7 +45,7 @@ const (
 	REGISTRY_USERNAME       ClusterInfoMapKey = "REGISTRY_USERNAME"       // registry username
 	REGISTRY_PASSWORD       ClusterInfoMapKey = "REGISTRY_PASSWORD"       // registry password
 	SOLDIER_ADDR            ClusterInfoMapKey = "SOLDIER_ADDR"            // soldier的地址
-	NETPORTAL_URL           ClusterInfoMapKey = "NETPORTAL_URL"           // netportal的集群入口url
+	CLUSTER_NAME            ClusterInfoMapKey = "CLUSTER_NAME"            // cluster_name
```
[the remark this answers is not in the document]
Comment (kakj-go) [reply apistructs/cluster_info.go:48]: Only the cluster information is passed, and the method is called as needed when the netportal address is actually required

**Review by sfwn — APPROVED** [review]

**Pull request merged**